### PR TITLE
Update Creatures and Scripts - Tanaris, Feralas, Silithus, Un' goro and Thousand Needles

### PR DESCRIPTION
**Pull request by NickTyrer**

### CODE REVIEW

**Comment by Gamemechanicwow** [comment]
> -- Kravel Koalbeard should kneel

Is this correct? Afaik Kravel is standing and only kneels when the quest Parts for Kravel is turned in.

**Comment by NickTyrer** [comment]
@Gamemechanicwow in classic when I checked he is kneeling. Also I checked wowhead's images of him and he is kneeling as well.

**Comment by Gamemechanicwow** [comment]
Hmm.. I checked some classic footage and he seems to be constantly kneeling. 

Wowhead has an image of him standing: https://wow.zamimg.com/uploads/screenshots/normal/17381-kravel-koalbeard.jpg
And he is standing in Joanas video
image

Here Joana turns in the Parts for kravel quest
https://www.youtube.com/watch?v=VBlnzRNTatI&feature=youtu.be&list=PL49Ga_-r7iKkxrbckm9CkUhWzxqrOnw-n&t=305
I know the resolution and quality is bad, but at 5:17 he is definitely kneeling as part of his script.

**Comment by NickTyrer** [comment]
@Gamemechanicwow I will leave him standing, classic seems to have a bad habit of breaking scripts.

File: sql/migrations/20201224122604_world.sql [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+INSERT INTO `generic_scripts` (`id`, `delay`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES (915701, 9, 3, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, -6202.634, -1114.64, -220.1, 0, 0, 'Bloodpetal Pest - Move');
+INSERT INTO `generic_scripts` (`id`, `delay`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES (915701, 12, 20, 1, 1, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 15, 0, 0, 0, 0, 'Bloodpetal Pest - Set Move type');
+
+
+-- Silithus
+-- Correct Cenarion Hold Infantry Patrols
+INSERT INTO `creature_groups` (`leader_guid`, `member_guid`, `dist`, `angle`, `flags`) VALUES
+(42897, 60006, 6, 3, 11);
```
Comment (ratkosrb) [inline sql/migrations/20201224122604_world.sql:183]: Guid 60006 is spawned on top of a big arc.

**Pull request merged**